tap_github/schemas/shared/owner.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Instead of the owner, the user name will suit this file more.

Reply:
Done